contentcuration/contentcuration/frontend/channelEdit/views/trash/TrashModal.vue

```diff
@@ -23,10 +23,10 @@
               <VLayout v-if="props.header.selectAll" row align-center>
                 <VFlex shrink>
                   <Checkbox
-                    :value="Boolean(selected.length)"
+                    :value="selected.length === items.length"
```

Comment:
To explicitly pass the boolean checkbox value, we need to use the prop `inputValue` instead. Without this, we have this issue: when all items are selected, the checkbox doesnt appear as selected.

https://github.com/learningequality/studio/assets/51239030/490b1e0c-82a0-485f-af02-c02de05bfad7

Reply:
Ah I see ! I'll make that change